src/test/resources/small.pdf.txt

```diff
@@ -0,0 +1 @@
+%PDF-foo
```

Comment:
Do we want to name this `small.pdf` instead as that is maybe more realistic that a PDF file would just have the `pdf` extension (not that our API technically knows about that when it receives the data).

I guess I was just a bit surprised to see `.txt` on the end but it isn't a big deal

Reply:
I did originally use the `.pdf` extension but needed to add the `.txt` extension to be able to edit it more easily in my IDE.  Also the file likely won't open in a reader anyway as this is just designed to get past the code in the client that checks if a file is really a PDF.

Reply:
Happy with that. I'll merge this!